components/tikv_util/src/metrics/threads_linux.rs

```diff
@@ -287,6 +289,139 @@ lazy_static! {
     };
 }
 
+#[inline]
+fn get_name(command: &str) -> String {
+    if command != "" {
+        return command.to_owned();
+    }
+    String::from("anony")
+}
+
+fn collect_metrics_by_name(
+    names: &HashMap<i32, String>,
+    values: &HashMap<i32, f64>,
```

Comment:
Can we use  HashMap<(i32, String), i64>?

Reply:
I think it may waste spaces if we store the thread names for each metric.